crates/storage/src/managers/mod.rs

```diff
@@ -1,2 +1,37 @@
+use std::sync::Arc;
+
+use strata_db::traits::Database;
+
 pub mod checkpoint;
 pub mod l2;
+
+/// A consolidation of database managers and ops. Ideally ops should not be exposed.
+#[derive(Clone)]
+pub struct DbManager {
+    l2_block_manager: Arc<l2::L2BlockManager>,
+    checkpoint_manager: Arc<checkpoint::CheckpointDbManager>,
+}
+impl DbManager {
+    pub fn l2(&self) -> Arc<l2::L2BlockManager> {
+        self.l2_block_manager.clone()
+    }
+
+    pub fn checkpoint(&self) -> Arc<checkpoint::CheckpointDbManager> {
+        self.checkpoint_manager.clone()
+    }
```

Comment:
Don't force these clones.

Reply:
What do you suggest? Returning refs?

Reply:
I could just return `&Arc<...>`

Reply:
Yeah that's the most straightforward way to do it.